<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectSourceDialog</class>
 <widget class="QDialog" name="SelectSourceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>189</width>
    <height>89</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MMpM</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Взять прошивку:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pb_from_file">
     <property name="text">
      <string>из файла</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pb_from_internet">
     <property name="text">
      <string>из интернета</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
